dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,32 @@
+public class Demo {
```

Comment:
The file has increased in 31 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._